### PR TITLE
Modal UI Enhancements

### PR DESCRIPTION
**Pull request by FinalAngel**
- [x] enhance title with more information
- [x] moved breadcrumb to top
- [x] adapted style / made it larger

image

### CODE REVIEW

File: cms/static/cms/js/modules/cms.sideframe.js [hunk 1]
```diff
@@ -54,7 +54,9 @@
                     that.close(true);
                 });
 
-                this.sideframe.find('.cms-sideframe-resize').bind('pointerdown.cms.sideframe', function (e) {
+                this.sideframe.find('.cms-sideframe-resize').on(
+                    'pointerdown.cms.sideframe contextmenu.cms.sideframe',
+                    function (e) {
                     e.preventDefault();
```
Comment (intellectronica) [inline cms/static/cms/js/modules/cms.sideframe.js]: Indentation

**Comment by intellectronica** [comment]
LGTM

File: cms/static/cms/js/modules/cms.sideframe.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.sideframe.js]: thats actually correct :)

Comment (intellectronica) [inline cms/static/cms/js/modules/cms.sideframe.js]: really? don't you indent within a block?

**Comment by vxsx** [comment]
LGTM

**Pull request merged**